src/query/expression/src/register.rs

```diff
@@ -4059,23 +4059,23 @@ fn erase_calc_domain_generic_5_arg<
 
 fn erase_function_generic_0_arg<O: ArgType>(
     func: impl Fn(&GenericMap) -> Result<Value<O>, String>,
-) -> impl Fn(&[ValueRef<AnyType>], &GenericMap) -> Result<Value<AnyType>, String> {
-    move |args, generics| func(generics).map(Value::upcast)
+) -> impl Fn(&[ValueRef<AnyType>], &GenericMap, usize) -> Result<Value<AnyType>, String> {
+    move |args, generics, _num_rows| func(generics).map(Value::upcast)
 }
 
 fn erase_function_generic_1_arg<I1: ArgType, O: ArgType>(
     func: impl Fn(ValueRef<I1>, &GenericMap) -> Result<Value<O>, String>,
-) -> impl Fn(&[ValueRef<AnyType>], &GenericMap) -> Result<Value<AnyType>, String> {
-    move |args, generics| {
+) -> impl Fn(&[ValueRef<AnyType>], &GenericMap, usize) -> Result<Value<AnyType>, String> {
```

Comment:
can we merge `&GenericMap, usize` into a new struct? Thus we can add more args into if we need them in the future.